src/main/java/com/coveo/k8sproxy/proxy/K8sReverseProxy.java

```diff
@@ -78,6 +79,47 @@
     private Timer timer = new Timer();
     private ReentrantReadWriteLock lock = new ReentrantReadWriteLock();
 
+    @RequestMapping(value = "/k8s_cluster_endpoint", method = RequestMethod.GET)
```

Comment:
I'm fine with the GET for the bookmark purposes but add the PUT here as well, you can pass an array to `method` here with both.